src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/__init__.py

```diff
@@ -0,0 +1,11 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-import
+
+import azure.cli.command_modules.keyvault.custom
+import azure.cli.command_modules.keyvault._params
+import azure.cli.command_modules.keyvault.generated
```

Comment:
>mport azure.cli.command_modules.keyvault [](start = 1, length = 40)

You should also import _help here since you added the file. This ensures that if anything is added, it will be applied.